client/cucumber/features/fabric-environments.feature

```diff
@@ -0,0 +1,130 @@
+        Then there should be a terminal open
+        Examples:
+        | nodeType       |
+        | fabric-peer    |
+        | fabric-ca      |
+        | fabric-orderer |  
+
+    Scenario Outline: It should persist data after being stopped
+        Given the Local Fabric is running
+        And the 'Local Fabric' environment is connected
+        And a <language> smart contract for <assetType> assets with the name <name> and version <version>
+        And the contract has been created
+        And the contract has been packaged
+        And the package has been installed
+        And the contract has been instantiated with the transaction '' and args '', not using private data
+        When I stop the Local Fabric
+        Then there should be a tree item with a label '<label>' in the 'Fabric Environments' panel
+        Then the tree item should have a tooltip equal to '<tooltip>'
+        When I start the Local Fabric
+        And the 'Local Fabric' environment is connected
+        Then there should be a instantiated smart contract tree item with a label '<instantiatedName>' in the 'Fabric Environments' panel
+        Examples:
+        | language   | assetType | name               | instantiatedName         | version | label                            | tooltip                                                                    |
+        | JavaScript | Conga     | JavaScriptContract | JavaScriptContract@0.0.2 | 0.0.2   | Local Fabric  ○ (click to start) | Creates a local development runtime using Hyperledger Fabric Docker images |
+
+    Scenario Outline: After tear down and start there are no smart contracts
+        Given the Local Fabric is running
+        And the 'Local Fabric' environment is connected
+        And a <language> smart contract for <assetType> assets with the name <name> and version <version> 
+        And the contract has been created
+        And the contract has been packaged
+        And the package has been installed
+        And the contract has been instantiated with the transaction '' and args '', not using private data
+        When I tear down the Local Fabric
+        Then there should be a tree item with a label 'Local Fabric  ○ (click to start)' in the 'Fabric Environments' panel
+        Examples:
+        | language   | assetType | name               | version |
+        | JavaScript | Conga     | JavaScriptContract | 0.0.2   | 
+
+
+    @otherFabric
+    Scenario: It should create an envrionment
+        When I create an environment 'myFabric'
+        Then there should be a tree item with a label 'myFabric' in the 'Fabric Environments' panel
+        And the tree item should have a tooltip equal to 'myFabric'
+
+    @otherFabric
+    Scenario Outline: It should connect to an environment
+        Given an environment 'myFabric' exists
+        And the wallet 'myWallet' with identity 'conga' and mspid 'Org1MSP' exists
+        When I connect to the environment 'myFabric'
```

Comment:
Should be be explicit with this step and make it something like:
`When I connect to the environment 'myFabric' with the wallet 'myWallet' and identity 'conga'`.

If we do something like that we can eventually add tests such as:

```
...
And the wallet 'myWallet' with identities 'conga1', 'conga2', 'conga3' and mspid 'Org1MSP' exists
When I connect to the environment 'myFabric' with the wallet 'myWallet' and identity 'conga2'
...
Then I should be connected to 'myFabric' with the identity 'conga2'
```

Reply:
don't need this